src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsMapping.java

```diff
@@ -32,10 +33,12 @@
 import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsRequestMapping.buildHearingRequestDetails;
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.EntityRoleCode.APPELLANT;
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.EntityRoleCode.APPOINTEE;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.EntityRoleCode.INTERPRETER;
```

Comment:
Could probably just * these

Reply:
dattipoeHMCTS, I think I had some of 'checkings' which told me to throw wildcards away